lib.py

```diff
@@ -333,6 +347,10 @@ def pay(self, amount_msat: int, binding = None):
                 except Exception as e:
                     self._plugin.log(f"Prism member keysend payment did not complete:  {e}", 'warn')
                     continue
+            elif m.destination == "":
```

Comment:
why would a member not have a desination? looks like Member.validate() checks that its a string

Reply:
oh I see in #107